.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,15 @@
+name: Run unit test
+on: push
```

Comment:
Thank you very much Brave!
we can test only on Pull Request 😃

Reply:
Thank you :))